src/python/grpcio/grpc/_adapter/_c/types/channel.c

```diff
@@ -164,7 +164,7 @@ PyObject *pygrpc_Channel_watch_connectivity_state(
   int last_observed_state;
   CompletionQueue *completion_queue;
   char *keywords[] = {"last_observed_state", "deadline",
-                      "completion_queue", "tag"};
+                      "completion_queue", "tag", NULL};
```

Comment:
Something went wrong this morning, and I looked up the error that I saw in the logs, saw an association with NULL missing from the called function's keyword arguments array, put this NULL in, and didn't see the error any more.

But now that I revert the change, I can't reproduce the error. I'll drop this file from the commit.